diff --git a/node_modules/@docusaurus/theme-classic/lib-next/theme/MDXComponents/index.js b/node_modules/@docusaurus/theme-classic/lib-next/theme/MDXComponents/index.js
index 42e4d82..42b9e8b 100644
--- a/node_modules/@docusaurus/theme-classic/lib-next/theme/MDXComponents/index.js
+++ b/node_modules/@docusaurus/theme-classic/lib-next/theme/MDXComponents/index.js
@@ -9,24 +9,24 @@ import Link from '@docusaurus/Link';
 import CodeBlock from '@theme/CodeBlock';
 import Heading from '@theme/Heading';
 const MDXComponents = {
-  code: (props) => {
-    const {children} = props;
+  // code: (props) => {
+  //   const {children} = props;
 
-    if (typeof children === 'string') {
-      if (!children.includes('\n')) {
-        return <code {...props} />;
-      }
+  //   if (typeof children === 'string') {
+  //     if (!children.includes('\n')) {
+  //       return <code {...props} />;
+  //     }
 
-      return <CodeBlock {...props} />;
-    }
+  //     return <CodeBlock {...props} />;
+  //   }
 
-    return children;
-  },
+  //   return children;
+  // },
   a: (props) => <Link {...props} />,
-  pre: (props) => {
-    const {children} = props;
-    return <CodeBlock {...children?.props} />;
-  },
+  // pre: (props) => {
+  //   const {children} = props;
+  //   return <CodeBlock {...children?.props} />;
+  // },
   h1: Heading('h1'),
   h2: Heading('h2'),
   h3: Heading('h3'),
diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/MDXComponents/index.js b/node_modules/@docusaurus/theme-classic/lib/theme/MDXComponents/index.js
index c17fb3c..31223b0 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/MDXComponents/index.js
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/MDXComponents/index.js
@@ -22,28 +22,28 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
  * LICENSE file in the root directory of this source tree.
  */
 const MDXComponents = {
-  code: props => {
-    const {
-      children
-    } = props;
+  // code: props => {
+  //   const {
+  //     children
+  //   } = props;
 
-    if (typeof children === 'string') {
-      if (!children.includes('\n')) {
-        return <code {...props} />;
-      }
+  //   if (typeof children === 'string') {
+  //     if (!children.includes('\n')) {
+  //       return <code {...props} />;
+  //     }
 
-      return <_CodeBlock.default {...props} />;
-    }
+  //     return <_CodeBlock.default {...props} />;
+  //   }
 
-    return children;
-  },
+  //   return children;
+  // },
   a: props => <_Link.default {...props} />,
-  pre: props => {
-    const {
-      children
-    } = props;
-    return <_CodeBlock.default {...children === null || children === void 0 ? void 0 : children.props} />;
-  },
+  // pre: props => {
+  //   const {
+  //     children
+  //   } = props;
+  //   return <_CodeBlock.default {...children === null || children === void 0 ? void 0 : children.props} />;
+  // },
   h1: (0, _Heading.default)('h1'),
   h2: (0, _Heading.default)('h2'),
   h3: (0, _Heading.default)('h3'),
